### ISSUE TITLE
Fix a11y issues of atoms components

### ISSUE THREAD

**Issue by Elohina** [buildit/gravity-ui-web#369]
After finishing the a11y test suite appeared different accessibility issues. In order to solve all the violations it would be better to go through them from lower complexity components (atoms) to higher.

This is the list of atom components that need to be improved:
- 05-icon-button
- 05-cite
- 08-toggle-menu
- 06-icon-toggle-button

**To Reproduce**
1. Go to `/gravity-ui-nunjucks`.
2. Run `npm run test:a11y`.
3. See the list of a11y violations.

**Expected behavior**
All of the violations listed have the information of what can be done in each case.
